src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32.cs

```diff
@@ -292,26 +290,24 @@ private static Win32 InternetShortcutProgram(string path)
             string[] lines = System.IO.File.ReadAllLines(path);
             string appName = string.Empty;
             string iconPath = string.Empty;
+            string urlPath = string.Empty;
             string scheme = string.Empty;
             bool validApp = false;
 
-            const string steamScheme = "steam";
+            Regex urlSchemes = new Regex(@"^steam:\/\/rungameid\/|^com\.epicgames\.launcher:\/\/apps\/");

```

Comment:
Is there any documentation about the format of the epic games url? If so, can you please share it?

Reply:
@alekhyareddy28 I wasn't able to find any official documentation on how epic games URL's are named. It is based mainly on the following [reddit ](https://www.reddit.com/r/EpicGamesPC/comments/dbzuxf/epic_launcher_creating_broken_shortcuts/)post and what I observed from installing multiple games on my desktop.